src/components/IconGrid.js

```diff
@@ -14,9 +15,11 @@ function IconGrid({ icons }) {
           <IconTile
             name={icon.name}
             title={`Download ${icon.name}.svg`}
-            onClick={() => {
-              download(icon.toSvg(), `${icon.name}.svg`, 'image/svg+xml')
+            onClick={e => {
               logDownload(icon.name)
+              return e.shiftKey
```

Comment:
Does this function need a `return`?

Reply:
I think a simple if/else might make sense here instead:
```jsx
if (event.shiftKey) {
  copy(icon.toSvg())
} else {
  download(icon.toSvg(), `${icon.name}.svg`, 'image/svg+xml')
}
```

Also, I usually prefer `event` over `e` for clarity :)

Reply:
Also, should we have a different google analytics event for "copy" or just group it into downloads?

Reply:
It might be nice to know how many people are using the feature 🤷‍♂️

Reply:
Yeah, that's probably a good idea. You can copy this file and change "download" to "copy": https://github.com/feathericons/feathericons.com/blob/master/src/utils/logDownload.js

Reply:
Then the if statement would look something like this: 
```js
if (event.shiftKey) { 
  copy(icon.toSvg())
  logCopy(icon.name)
} else {
  download(icon.toSvg(), `${icon.name}.svg`, 'image/svg+xml')
  logDownload(icon.name)
}
```